contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -2,62 +2,159 @@ category: application
 doc: |
   This is the most general application definition for multidimensional
   photoelectron spectroscopy.
+  
+  Groups and fields are named according to the
+  `ISO 18115-1:2023`_ specification as well as the `IUPAC Recommendations 2020`_.
+  
+  .. _ISO 18115-1:2023: https://www.iso.org/standard/74811.html
+  .. _IUPAC Recommendations 2020: https://doi.org/10.1515/pac-2019-0404
+symbols:
+  doc: |
+    The symbols used in the schema to specify e.g. dimensions of arrays
+  n_transmission_function: |
+    Number of data points in the transmission function.
 type: group
 NXmpes(NXobject):
   (NXentry):
+    definition:
+      \@version:
+      enumeration: [NXmpes]
     title:
     start_time(NX_DATE_TIME):
       doc: |
         Datetime of the start of the measurement.
-    definition:
-      \@version:
-      enumeration: [NXmpes]
+        Should be a ISO8601 date/time stamp. It is recommended to add an explicit time zone,
+        otherwise the local time zone is assumed per ISO8601.
+    end_time(NX_DATE_TIME):
+      exists: recommended
+      doc: |
+        Datetime of the end of the measurement.
+        Should be a ISO8601 date/time stamp. It is recommended to add an explicit time zone,
+        otherwise the local time zone is assumed per ISO8601.
+    method:
+      exists: recommended
+      doc: |
+        A name of the experimental method according to `Clause 11`_ of
+        the `ISO 18115-1:2023`_ specification.
+        
+        Examples include:
+            * X-ray photoelectron spectroscopy (XPS)
+            * angle-resolved X-ray photoelectron spectroscopy (ARXPS)
+            * ultraviolet photoelectron spectroscopy (UPS)
+            * angle-resolved photoelectron spectroscopy (ARPES)
+            * hard X-ray photoemission spectroscopy (HAXPES)
+            * near ambient pressure X-ray photoelectron spectroscopy (NAPXPS)
+            * photoelectron emission microscopy (PEEM)
+            * electron spectroscopy for chemical analysis (ESCA)
+        
```

Comment:
Should we include types here not covered by this ISO? E.g. time-resolved ARPES, spin-ARPES, momentum microscopy, etc.?

Reply:
Ok, but then I need to change the whole docstring. I went with:
```yaml
Name of the experimental method.

If applicable, this name should match the terms given by `Clause 11`_ of
the `ISO 18115-1:2023`_ specification.

Examples include:
    * X-ray photoelectron spectroscopy (XPS)
    * angle-resolved X-ray photoelectron spectroscopy (ARXPS)
    * ultraviolet photoelectron spectroscopy (UPS)
    * angle-resolved photoelectron spectroscopy (ARPES)
    * hard X-ray photoemission spectroscopy (HAXPES)
    * near ambient pressure X-ray photoelectron spectroscopy (NAPXPS)
    * photoelectron emission microscopy (PEEM)
    * electron spectroscopy for chemical analysis (ESCA)
    * time-resolved angle-resolved X-ray photoelectron spectroscopy (trARPES)
    * spin-resolved angle-resolved X-ray photoelectron spectroscopy (spin-ARPES)
    * momentum microscopy
   ```